test/blockchaintest/blockchaintest_runner.cpp

```diff
@@ -29,6 +29,9 @@ struct TransitionResult
     state::BloomFilter bloom;
     int64_t blob_gas_left;
     TestState block_state;
+
+    /// The execution of system contracts has failed what makes the block invalid.
+    bool system_contracts_failed = false;
```

Comment:
We didn't support other invalid block tests before? Maybe makes sense to name this more general "invalid_block" ?

Reply:
Ah I see other invalid cases are handled differently, so this is fine I guess

Reply:
It seems the core functionality of t8n is to partition transactions into valid / rejected...